### DISCUSSION TITLE
Mic and camera don't work in Google Meet

### DISCUSSION THREAD

**Discussion by gregadro** [zen-browser/desktop#1548] [General]
After the last update (1.0.1-a.2 Win 11) the microphone and camera don't work in Google Meet. Tried to reinstall the browser, and reassign the permissions but it didn't fix. Any suggestions?

Thanks.

**Comment by AtzeDeVries** [comment]
same for me for teams / slack etc. This is on mac. I need to fully test this, but it seems that the internal webcam works but external camera (via Camo) does not.

**Comment by gregadro** [comment]
This has happened only on my laptop. On a desktop computer, it works fine. Both are Windows 11.

**Comment by gregadro** [comment]
With 1.0.1-a.4 everything works fine now.  🙌

**Comment by AndreJorgeLopes** [comment]
This seems to still be a problem in version 1.0.1-a.19.

**Comment by alxlion** [comment]
Still have problem with camera and micro on Google Meet with version 1.13.2b

**Comment by adrianescat** [comment]
Hello! I have that problem too. Not with all my mic options, tho. It works for my iPhone, the camera mic (which works), and the Mac mic. But not the mic connected to my Focusrite Scarlett 2i2. I gave it permissions, I closed Zen, and opened it again. It doesn't work.

Everything works on other browsers.

**Comment by chakraskun** [comment]
yeah, the problem still persist on version 1.17.2b, works on other browser (firefox & chrome)

**Reply by raufoon** [reply]
My PC shuts down when trying to connect mic and camera on google meet in 1.17.12b (Firefox 145.0.2) (64-bit), Windows 11.
